### ISSUE TITLE
Learning rates do not match paper

### ISSUE THREAD

**Issue by bsaund** [Yang7879/3D-RecGAN-extended#4]
Thanks for the work and for releasing the code! We hope to use this on our robot.
I also appreciate how well all of the learning parameters are documented.

In the paper, Section "3.4 Training" lists the following learning rates for the Adam optimizer:
discriminator: 0.0001
generator: 0.0005

The code provided uses:
discriminator: 0.00005
generator: 0.0001 

Is this a typo (both switching dis and gen, and missing a '0')? Did you change the learning rates after writing the paper?

**Comment by Yang7879** [comment]
@bsaund  Great thanks for reporting the mismatch. The learning rate in the release code is correct.  We will update it in next paper version.
